### DISCUSSION TITLE
Galaxy Brain achievement

### DISCUSSION THREAD

**Discussion by imanoar** [tarikmanoar/tarikmanoar#4] [Q&A] [answered]
How to get it?

**Comment by tarikmanoar** [comment, marked as answer]
We currently assume it's
"You need to become the marked answer to 2 discussions?"
but it seems like there are more conditions, for example, you don't seem to be able to mark yourself as the answer.

But I invite you to select this as the answer, because then I should have answered 2 discussions (which I didn't create myself) and be marked by the OP as the answer, soooo that should yield Galaxy Brain, hopefully.

**Comment by Ye-Yint-Nyo-Hmine** [comment]
Could you mark this as an answer?

**Reply by tarikmanoar** [reply]
Start a new discussion I already made it
